what-is-the-ic/2_nns/index.md

```diff
@@ -4,12 +4,6 @@ card: /img/what-is-the-ic/nns.jpg
 cardImageFit: cover
 ---
 
-As its governance system, the Network Nervous System (NNS) provides crucial functionality to the IC.
-
-The NNS holds the ICP ledger, which tracks the accounts and transactions of the ICP utility token.
-It enables token holders to stake ICP tokens and thereby participate in voting on every aspect of the IC. For example, stakers may vote to upgrade the IC software or vote to extend the IC by adding a new subnet and so forth (learn more about staking, voting, and rewards [here](https://wiki.internetcomputer.org/wiki/Staking,_voting_and_rewards)).
-Furthermore, the NNS maintains the entire IC structure, such as information about all nodes or their allocation to subnets. It constitutes a single, but decentralized, source of truth.
-
-Moreover, the NNS provides the means to power computation on the IC by converting ICP tokens to *cycles*, which are consumed when running applications.
-
-The NNS is the world’s most advanced decentralized autonomous organization (DAO). Learn more about the NNS [here](https://internetcomputer.org/how-it-works/network-nervous-system-nns/).
+The [Network Nervous System (NNS)](/how-it-works/1_about/08_network-nervous-system-nns.subpage.md) is the decentralized source of truth regarding the whole IC structure, storing information about all nodes and their allocation to subnets.
```

Comment:
any chance you can remove the `.subpage` part in the linked url? It's quite ugly but no big deal

Reply:
We wanted to link explicitly to the subpage rather than nns.card.md

Reply:
What is subpage.md? Apparently, `https://internetcomputer.org/how-it-works/1_about/08_network-nervous-system-nns.subpage.md` is not a proper link...